### PR TITLE
Add actual after Expecting to disambiguate error messages

### PR DESCRIPTION
**Pull request by utoniumharsha**
Check List:
Fixes nothing
Unit tests : Updated
Javadoc with a code example (on API only) : NA

### CODE REVIEW

File: src/main/java/org/assertj/core/error/ShouldHaveBinaryContent.java [hunk 1]
```diff
@@ -56,17 +56,17 @@ public static ErrorMessageFactory shouldHaveBinaryContent(InputStream actual, Bi
   }
 
   private ShouldHaveBinaryContent(File actual, BinaryDiffResult diff) {
-    super("%nFile:%n  %s%ndoes not have expected binary content at offset %s, expecting:%n  %s%nbut was:%n  %s",
+    super("%nFile:%n  %s%ndoes not have expected binary content at offset %s, expecting actual:%n  %s%nbut was:%n  %s",
```
Comment (joel-costigliola) [inline src/main/java/org/assertj/core/error/ShouldHaveBinaryContent.java:59]: revert the 3 changes in this file, I realize adding `actual` does not improve the error messages in these cases

**Review by joel-costigliola — CHANGES_REQUESTED** [review]
thanks, now you need to update the unit tests ;-)

File: src/main/java/org/assertj/core/error/ShouldHaveBinaryContent.java (hunk 1 above)
Comment (utoniumharsha) [reply src/main/java/org/assertj/core/error/ShouldHaveBinaryContent.java]: Yes, agreed. Reverted changes for this file.

**Comment by joel-costigliola** [comment]
Integrated thanks @utoniumharsha!

**Pull request merged**